isthmus/src/test/resources/tpcds/README.md

```diff
@@ -0,0 +1,43 @@
+
+## Setup of Schema and Queries
+- Schema using `org.apache.calcite.adapter.tpcds.TpcdsSchema` from `org.apache.calcite:calcite-plus:1.28.0`
+- For queries started with `net.hydromatic.tpcds.query.Query` and then fixed generation issues
+  using Spark SQL tpcds benchmark queries.
+
+## Status of testing on 6/26/2022
```

Comment:
I'd recommend filing github issues (one per bullet point below), then removing this section as it'll quickly get out of date (and be a pain to keep in sync).

Reply:
Added